xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/extension.vm

```diff
@@ -613,10 +613,12 @@ $namespace##
     #set ($dependencyName = "<a href=""$dependencyURL"" class=""extension-link"">#displayExtensionName($dependencyExtension)</a>")
   #end
   <div class="dependency-item extension-item-$dependencyStatus">
-    <span class="extension-name">${dependencyName}</span><span class="extension-version">$!dependencyVersion</span>
-    #if ($extensionNamespace.startsWith('wiki:'))
-      <span class="extension-namespace">$services.localization.render('extensions.info.dependency.wiki', ["#wikiHomePageLink($extensionNamespace)"])</span>
-    #end
+    <div class="dependency-item-title">
```

Comment:
Why do you need to group these? It goes against the extension display where the name, version and status are siblings. I think we should be consistent: if we keep them as siblings in the extension display we should do the same in the extension dependency display. Basically, in both cases we have a "line" that shows extension name, version, name space (optional) and status (optional).

Reply:
Indeed, using siblings in both places will be more consistent, I updated the structure accordingly in the latest commit, which results in the display below with default font / large font. Many thanks for your guidance.

![extension-dependency](https://user-images.githubusercontent.com/1022195/117150280-48afca80-adb8-11eb-9eeb-b1ad6fbaae8c.jpg)

![extension-dependency-large-font](https://user-images.githubusercontent.com/1022195/117150284-49486100-adb8-11eb-8e02-5b14961111b1.jpg)